tools/downloader/topologies_test.py

```diff
@@ -0,0 +1,116 @@
+import unittest
+import os
+import numpy as np
+import cv2 as cv
+
+import topologies as omz
+
+from pathlib import Path
+from parameterized import parameterized
+from openvino.inference_engine import IECore
+from openvino import inference_engine
+
+class TestTopologies(unittest.TestCase):
+    @parameterized.expand(['FP16', 'FP32'])
+    def test_load_intel_without_precision(self, p):
+        face_detection = omz.Topology('face-detection-0100')
+        xml_path, bin_path = face_detection.get_ir(precision=p)
+        
+        self.assertTrue(os.path.exists(xml_path))
+        self.assertTrue(os.path.exists(bin_path))
+
+        self.assertEqual(os.path.basename(xml_path), 'face-detection-0100.xml')
+        self.assertEqual(os.path.basename(bin_path), 'face-detection-0100.bin')
+
+        self.assertEqual(os.path.basename(os.path.dirname(xml_path)), p)
+        self.assertEqual(os.path.basename(os.path.dirname(bin_path)), p)
+
+    def test_load_intel_FP32(self):
+        face_detection = omz.Topology('face-detection-0100', precision='FP32')
```

Comment:
Why not to combine `test_load_intel_FP32` with `test_load_intel_without_precision`?